pages/blog/whats-new-in-graphql-codegen-v2.mdx

```diff
@@ -0,0 +1,135 @@
+import withArticle from '../../ui/blog/article';
+
+export const meta = {
+  title: "What's new in GraphQL-Codegen v2?",
+  author: 'dotan',
+  tags: ['graphql', 'codegen', 'typscript'],
+  date: '2021-08-03',
+  description:
+    'Human-readable types, improved infrastructure, a new plugin for TypeScript and more!',
+  image: '/blog-assets/graphql-code-generator/cover.png',
+};
+
+export default withArticle({ ...meta });
+
+[GraphQL Code Generator](https://github.com/dotansimha/graphql-code-generator/) is here for ~5 years, and it's improving developer exprience for many developers (>3M downloads a month on NPM!)
+
```

Comment:
We have constantly listening and gradually improving Codegen throughout all these years